<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result</class>
 <widget class="QDialog" name="Result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Result{
background-color: rgb(203, 214, 218)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 50pt &quot;Yu Gothic UI Light&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Рівень пройдено!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
font: 25 28pt &quot;Yu Gothic UI Light&quot;;
color: rgb(45, 44, 44);
border-radius: 15px;
border-color: white;
border: 2px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="homeButton">
         <property name="minimumSize">
          <size>
           <width>396</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#homeButton{
background-color: rgb(250, 191, 35);
color: white;
font: 15pt &quot;Yu Gothic UI Semilight&quot;;
border:0px;}

#homeButton:hover{
background-color:rgb(74, 72, 73);
color:white;
font: 14pt &quot;Yu Gothic UI Semilight&quot;;}</string>
         </property>
         <property name="text">
          <string>На головну</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextLevButton">
         <property name="minimumSize">
          <size>
           <width>396</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#nextLevButton{
background-color: rgb(250, 191, 35);
color: white;
font: 15pt &quot;Yu Gothic UI Semilight&quot;;
border:0px;}

#nextLevButton:hover{
background-color:rgb(74, 72, 73);
color:white;
font: 14pt &quot;Yu Gothic UI Semilight&quot;;}</string>
         </property>
         <property name="text">
          <string>Наступний рівень</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>200</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/res/img/title.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
